.github/workflows/release-tags.yml

```
@@ -64,19 +65,22 @@ jobs:
 
       - name: Release under current tag
         run: |
+          PRERELEASE=""
+          if [[ $SNAPSHOT ]]; then
```

This will always be true, because `if [[ non-empty string ]]` is true. You want:
```suggestion
          if [[ "$SNAPSHOT" == "true" ]]; then
```